README.md

```diff
@@ -8,24 +8,53 @@ Built on [bitjson/typescript-starter](https://github.com/bitjson/typescript-star
 
 ## Usage
 
-The client needs to be configured with authentication tokens.
+The client needs to be configured with client credentials.
 Please contact engineers@permanent.org for access.
 
 ```js
 // ES import
-import { Permanent } from '@permanentorg/node-sdk';
+import { PermanentOAuthClient } from '@permanentorg/node-sdk';
 
 // CommonJS
-const Permanent = require('@permanentorg/node-sdk').Permanent;
+const Permanent = require('@permanentorg/node-sdk').PermanentOAuthClient;
 
 // Configure with credentials
-const permanent = new Permanent({
-  sessionToken,
-  mfaToken,
-  archiveNbr,
+const client = new PermanentOAuthClient(
+  clientId,
+  clientSecret,
   baseUrl, // optional, defaults to prod environment URL
```

Comment:
Oh, you're right. That change was to accommodate [Axios](https://axios-http.com/docs/req_config), which requires that capitalization. I accomplish that renaming via a parameter name, which clients do not explicitly set, only see: https://github.com/PermanentOrg/node-sdk/blob/0f7f35df5caf9b28343da3e5bbfd69b755f8e850/src/lib/api/api.service.ts#L50-L55

The object passed to the `ApiService` constructor is used directly in instantiating an Axios instance:
https://github.com/PermanentOrg/node-sdk/blob/0f7f35df5caf9b28343da3e5bbfd69b755f8e850/src/lib/api/api.service.ts#L33-L35

I think changing the capitalization to `baseURL` in the API we expose to clients would be a breaking change, and if we do not then we need to rename from `baseUrl` to `baseURL` somewhere. I could make `fromToken` and `fromSession` take lowercase `baseUrl`, and then change the object construction to `baseURL: baseUrl`; that would only be in private code rather than exposed to the user at all.

Which I think is what you might have been suggesting in the first place; sorry it took me a moment to get there. :sweat_smile:  Yes, good call!

Reply:
Ha, no, I did not realize the complexity of that change, thanks for the explanation!  I don't think it's necessary but it would be welcome.

Reply:
After taking a pass at making this change, I think I'm going to change my mind. The argument to `fromToken` and `fromSession` is internal only, and not user facing - no users ever construct an `ApiService` instance directly. The inconsistency is always going to exist, because our user-facing API takes an object with a `baseUrl` key, and Axios takes a configuration with a `baseURL` key. I don't know of a way to make it cleaner than it is - I think this is as good as it gets!